link/src/main/java/com/stripe/android/link/ui/LinkButtonView.kt

```diff
@@ -43,8 +43,8 @@ import com.stripe.android.link.theme.linkColors
 
 private val LinkButtonVerticalPadding = 6.dp
 private val LinkButtonHorizontalPadding = 10.dp
```

Comment:
I think these were different because of the rounded shape. Should we make them equal now that we have a rectangular button shape (as with all the other questions: @jarvis-stripe, what do you think?)